app/controllers/expenses_controller.rb

```diff
@@ -27,7 +31,8 @@ def update
     if @expense.update_attributes expenses_params
       redirect_to user_path(session[:user_id])
     else
-      render :partial => 'errors', flash: { error: "Your expense must be greater than $0."}
+      flash[:error] = "Your expense must be greater than $0."
+      redirect_to user_path(session[:user_id])
```

Comment:
suggestion - since both if/else statements drive to the same path, can you move this redirect to after line 36?